config/betterquesting/DefaultQuests/Quests/AndSoItBegins-AAAAAAAAAAAAAAAAAAAAAA==/YouReap-AAAAAAAAAAAAAAAAAAAABg==.json

```diff
@@ -31,7 +31,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "You won\u0027t always get food rewards from this book, so it might be a good idea to take care of that problem. As you\u0027ve created a hoe already, go and find some water nearby and make a field.  With this modpack you can only make farmland within range of water. Let\u0027s trade some sticks for a carrot, so you can start your own farm.\n\nTo get other wheat, use a hoe on grass away from water. You\u0027ll get Witchery seeds too, save those for later. Start planting a cotton field, it\u0027s a key early game resource.\n\n§3PS: You can also grab a bunch from villages, and also grow the seeds from Pam\u0027s garden food."
+      "desc:8": "You won\u0027t always get food rewards from the quests, so it might be a good idea to take care of it.\n \nAs you\u0027ve crafted a hoe before, go and find some water nearby to make a field.  With this modpack you can only make farmland within range of water. Let\u0027s trade some sticks for a carrot, so you can start your own farm.\n\nTo get other wheat, use a hoe on grass away from water. You\u0027ll get Witchery seeds too - save those for later. \n\nStart planting a Cotton field because it\u0027s a key early-game resource.\n\nYou can also get seeds from villages, and grow the seeds from the Pam\u0027s garden food."
```

Comment:
"that problem" was better than "it". "already" was better than "before".

Reply:
Thank you! Fixed.